AgentOntology.ttl

```diff
@@ -1300,6 +1365,47 @@ cco:ContractorRole rdf:type owl:Class ;
                    rdfs:label "Contractor Role"@en .
 
 
+###  http://www.ontologyrepository.com/CommonCoreOntologies/Country
+cco:Country rdf:type owl:Class ;
+            rdfs:subClassOf cco:GeopoliticalEntity ;
+            dcterms:creator "https://cubrc.org"^^xsd:anyURI ;
+            cco:definition "A Government Domain that is associated with certain distinct political, ethnic, or cultural characteristics, and which delimits a Government that has legitimate authority over a Populace."@en ;
+            cco:is_curated_in_ontology "http://www.ontologyrepository.com/CommonCoreOntologies/Mid/AgentOntology"^^xsd:anyURI ;
+            rdfs:label "Domain of a Country"@en ;
+            skos:editorialNote "'Country' is an unclear term. Does it refer to the people, the government, the site, or the material in the site? Thus, instead of defining 'Country' we have defined four country elements found under the class 'Country Element'. We leave it to the users to define 'Country' as it suites their purposes."@en ,
+                               "What is properly referred to as a 'Country' is highly dependent on the classification practices of the international community. For example: The United Kingdom is a sovereign country that has Wales, a non-sovereign Country, as a member. Vatican City is a sovereign country that is landlocked by a city: Rome, Italy. Puerto Rico is not a county (it's a territory) but has a government, a distinct culture, and so on."@en ;
+            skos:prefLabel "Domain of a Country"@en .
+
+
+###  http://www.ontologyrepository.com/CommonCoreOntologies/CountryElement
+cco:CountryElement rdf:type owl:Class ;
+                   owl:equivalentClass [ rdf:type owl:Class ;
```

Comment:
Is this not a disjoint union?

Reply:
It is disjoint insomuch as this would be inherited by the classes from their parents classes. Thus, not 'DisjointUnionOf:'. Also, I've added more editorial notes to help clarify the intended use of these terms.